hw/ip/otbn/dv/model/iss_wrapper.h

```diff
@@ -99,6 +99,9 @@ struct ISSWrapper {
   // Mark all of IMEM as invalid so that any fetch causes an integrity error.
   void invalidate_imem();
 
+  // Mark all of IMEM as invalid so that any fetch causes an integrity error.
```

Comment:
This should be DMEM (not IMEM) and you want "so that any load causes" (rather than "fetch")